utils/kubernetes/describe/describe_test.go

```diff
@@ -0,0 +1,137 @@
+package describe
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/jarcoal/httpmock"
+	meshkitkube "github.com/layer5io/meshkit/utils/kubernetes"
+)
+
+type MockClient struct {
+	Object       string
+	Method       string
+	RequestUrl   string
+	ResponseCode int
+	Response     string
+}
+
+func TestDescribe(t *testing.T) {
+	ctx := context.Background()
+	//http time out to handle request and response
+	timeout := 300 * time.Second
+	clienttime := &http.Client{
+		Timeout:   timeout,
+		Transport: http.DefaultTransport,
+	}
+	httpmock.ActivateNonDefault(clienttime)
+	//set up mock client  to handle api request and return expected responses
+	mckpod := MockClient{
+		Object:       "Pod",
+		Method:       "GET",
+		RequestUrl:   fmt.Sprintf("%sapi/v1/namespaces", ctx),
```

Comment:
@Philip-21 You dont need to use `ctx` here as the prefix. Use the address given in test.yml